src/clear-all-refinements/index.ts

```diff
@@ -0,0 +1,19 @@
+import { alias, tag, Tag } from '@storefront/core';
+
+@alias('clearAllRefinements')
```

Comment:
I wouldn't give this component an alias as it only has one direct child and so doesn't need its alias available in a nested child